### PR TITLE
[sessions]: Enforce mutual exclusion between user and agent mentions on paste

### PR DESCRIPTION
**Pull request by smb2268**
## Description
This expands on the PR I put up to strip excess agent mentions on paste, also taking into account user mentions. It respects whichever is pasted first (user or first agent) and strips the rest of the mentions. Populates notification text based on what we stripped out and why 

## Tests
Test via link! Testing no regression on the behavior already merged (agent only paste removal) as well as new user paste scenarios

## Risk
Low, these are all edge case 

## Deploy Plan

- [ ] deploy front

### CODE REVIEW

File: front/components/assistant/conversation/input_bar/InputBarContainer.tsx [hunk 1]
```diff
@@ -417,14 +418,32 @@ const InputBarContainer = ({
...
+        let description: string;
+        switch (payload.reason) {
+          case "extra-agents":
+            title = "Agent mentions removed";
+            description = `${payload.count} ${payload.count === 1 ? "agent mention was" : "agent mentions were"} removed. Only one agent can be used at a time.`;
+            break;
+          case "users-stripped-for-agent":
+            title = "User mentions removed";
+            description =
+              "You can't mention both users and agents in the same message.";
```
Comment (ykmsd) [inline front/components/assistant/conversation/input_bar/InputBarContainer.tsx]:
```suggestion
              "You can’t mention both users and agents in the same message.";
```

File: front/components/assistant/conversation/input_bar/InputBarContainer.tsx [hunk 2]
```diff
@@ -417,14 +418,32 @@ const InputBarContainer = ({
...
+            break;
+          case "user-conflict":
+          case "mixed-conflict":
+            title = "Agent mentions removed";
+            description =
+              "You can't mention both users and agents in the same message.";
```
Comment (ykmsd) [inline front/components/assistant/conversation/input_bar/InputBarContainer.tsx]:
```suggestion
              "You can’t mention both users and agents in the same message.";
```

**Review by ykmsd — APPROVED** [review]
